### DISCUSSION TITLE
Struggling with Skin Realism in Fooocus

### DISCUSSION THREAD

**Discussion by orangejuiceone** [lllyasviel/Fooocus#3254] [General]
Hi everyone,

I recently installed Fooocus on my PC (GPU RTX3080) following the provided instructions. I also checked out guides on how to use prompts, like "sharply focused human skin" and "visible facial skin pores." Despite this, the skin realism in my generated images is lacking. I enabled negative prompts as well, such as "perfect skin," "unrealistic," "bad," "ugly," and "low-quality," but the image generation, skin texture, and overall quality still aren't up to par.

When I compare my results with Midjourney, the difference is like night and day. I like the  Fooocus' feature of using existing images along with prompts and the fine control options, but so far, my outcomes have been disappointing.

Any ideas on what I might be doing wrong or tips to significantly improve the skin realism output? Thanks in advance!

**Comment by himpundir** [comment]
Try these settings and you should be getting the desired results:
Option 1:
Base Model: juggernautXL_v8Rundiffusion.safetensors
Refiner at (0.4 switch): realisticVisionV60B1_v51VAE
Refiner  option 2: epicphotogasm_ultimateFidelity.safetensors

**Reply by orangejuiceone** [reply]
Thanks for sending over the settings @himpundir these helped for sure!
